source/views/transportation/bus/bus-view.js

```diff
@@ -1,66 +1,117 @@
 // @flow
 
 import React from 'react'
-import {ScrollView} from 'react-native'
+import {ScrollView, RefreshControl} from 'react-native'
 import type {BusLineType} from './types'
 import {BusLine} from './bus-line'
 import moment from 'moment-timezone'
 import {NoticeView} from '../../components/notice'
 import type {TopLevelViewPropsType} from '../../types'
+import delay from 'delay'
+import {reportNetworkProblem} from '../../../lib/report-network-problem'
 
-import {data as defaultBusLines} from '../../../../docs/bus-times.json'
+import * as defaultData from '../../../../docs/bus-times.json'
 
 const TIMEZONE = 'America/Winnipeg'
 
-export class BusView extends React.PureComponent {
-  static defaultProps = {
-    busLines: defaultBusLines,
-  }
+const GITHUB_URL = 'https://stodevx.github.io/AAO-React-Native/bus-times.json'
```

Comment:
Can the repo URL get turned into a constant somewhere that this is built off of?  (This is out-of-scope for this PR but an idea nonetheless)

Reply:
I keep pushing that back to when I want to centralize all data fetching in the redux store… this is, however, unlikely to change, and is easy to grep for.